### PR TITLE
spotdl: 4.0.7 -> 4.1.3

### PR DESCRIPTION
**Pull request by ckiee**
###### Description of changes

A patch and another missing dependency later, it works again.

Ref https://github.com/spotDL/spotify-downloader/issues/1789
Changelog https://github.com/spotDL/spotify-downloader/compare/v4.0.7...v4.1.3

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage).<br><br> **Almost OOM'd my system while writing this, so no.**
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/ytmusicapi/default.nix [hunk 1]
```diff
@@ -37,7 +37,6 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python API for YouTube Music";
     homepage = "https://github.com/sigma67/ytmusicapi";
-    changelog = "https://github.com/sigma67/ytmusicapi/releases/tag/${version}";
```
Comment (dotlambda) [inline pkgs/development/python-modules/ytmusicapi/default.nix:40]: It actually does provide a changelog in this case.

**Review by dotlambda — COMMENTED** [review]
Can we update the pykakasi URLs to https://codeberg.org/miurahr/pykakasi ?

**Comment by dotlambda** [comment]
> backport release-22.11

Why?

**Comment by ckiee** [comment]
The comment in yt-dlp applies to spotdl too:
```nix
  # The websites yt-dlp deals with are a very moving target. That means that
  # downloads break constantly. Because of that, updates should always be backported
  # to the latest stable release.
```
…and it is why I decided to go out and update spotdl in the first place. :P

**Comment by ckiee** [comment]
Force pushed to add a similar notice about the backport to the spotdl deriation.

File: pkgs/development/python-modules/ytmusicapi/default.nix (hunk 1 above)
Comment (ckiee) [reply pkgs/development/python-modules/ytmusicapi/default.nix:40]: Please verify it is the correct URL now

**Comment by wegank** [comment]
Closed by #228835.

**Pull request closed without merging**